filesystem/usr/local/share/landoop/sample-data/group_logviewer.yaml

```diff
@@ -0,0 +1,17 @@
+name: logviewer
+description: "logviewer"
+applicationPermissions:
+- ViewKafkaConsumers
+- ViewConnectors
+- ViewSQLProcessors
+- ViewSchemaRegistry
+- ViewTopology
+dataNamespaces:
+- wildcards:
+    - "*"
+  permissions:
+    - QueryTopic
+    - ShowTopic
+    - ViewSchema
+  system: Kafka
+  instance: Dev
```

Comment:
If it's supported, can we add a description?
> This group only lets you view topics starting with log. To use it, try to login as the `logviewer` with password `logviewer`.

Reply:
Added description